packages/sql-integrations/src/sql-integration-auth-methods.ts

```diff
@@ -0,0 +1,31 @@
+export const AwsAuthMethods = {
+  AccessKeys: 'access-keys',
+  IamRole: 'iam-role',
+} as const
+
+export type AwsAuthMethod = (typeof AwsAuthMethods)[keyof typeof AwsAuthMethods]
+
+export const SnowflakeAuthMethods = {
```

Comment:
The AWS methods were not grouped in the same module as the other auth methods, I moved it here. Also, I unified the casing of the keys in all these maps/enums to be PascalCase.